src/Database/PostgreSQL/PQTypes/Model/Trigger.hs

```diff
@@ -51,13 +62,17 @@ data TriggerEvent
 
 -- | Trigger.
 --
--- @since 1.15.0.0
+-- @since blah blah blah
```

Comment:
This shouldn't be modified, it's the creation date, not modification date.

Reply:
done